src/utils/time.ts

```diff
@@ -143,6 +143,20 @@ export const formatHourAndMinute = (utcSeconds: number) => {
   return hh + ":" + mm;
 };
 
+export const formatHour = (utcSeconds: number) => {
+  const date = new Date(utcSeconds * ONE_SECOND_IN_MILLISECONDS);
+  const hh = String(date.getHours()).padStart(2, "0");
+  return hh;
+};
+
```

Comment:
What is the expected functionality of this function?
I wounder if you could use fsn lib to format numbers?

Reply:
> I wonder if you could use fsn lib to format numbers?

@mike-lvov Do you mean `date-fns` here? If so, agreed, many useful helpers depending on the specific needs. eg.

- https://date-fns.org/v2.20.1/docs/format
- https://date-fns.org/v2.20.1/docs/formatDistance
- https://date-fns.org/v2.20.1/docs/formatDuration
- https://date-fns.org/v2.20.1/docs/formatRelative

Reply:
@sunny-viktoryia have you seen this comment?

Reply:
removed the function as it is no longer used https://github.com/sparkletown/sparkle/pull/1302/commits/32efd4933865e151ac1c91f8642b9ca568bdf7c8